### DISCUSSION TITLE
Zstd compression level

### DISCUSSION THREAD

**Discussion by jbowlen** [pacman82/odbc2parquet#272] [General]
Hi,

first of all thanks for this neat tool :)

I have a small question: which compression level does zstd use, as it seems that one cannot give that information as an input value. 

KR

**Comment by pacman82** [comment]
Hello @jbowlen ,

thanks for the kind words about `odbc2parquet`.

The compression level for zstd is hardcoded to `1` in the upstream `parquet` crate used by this `odbc2parquet`. See: <https://github.com/apache/arrow-rs/blob/0268bba4c01c2b83986c023258ad4405c29cabff/parquet/src/compression.rs#L300>

Best, Markus
